include/mbedtls/rsa.h

```diff
@@ -736,6 +736,36 @@ int mbedtls_rsa_rsaes_oaep_decrypt( mbedtls_rsa_context *ctx,
                             unsigned char *output,
                             size_t output_max_len );
 
+/**
+ * \brief          Encode a hash into a DigestInfo structure as specified
+ *                 by PKCS#1(RFC 8017, EMSA-PKCS1-v1_5-ENCODE step 2).
```

Comment:
Minor: space before the open parenthesis

Reply:
Done.